docs/source/docker/advanced/active_learning.rst

```diff
@@ -1,7 +1,294 @@
+
+.. note::
+    
+    Box coordinates are floats measured from the top left image corner (and are 0-indexed).
+    We recommend rounding coordinates to the nearest tenth of a pixel to reduce resulting JSON file size.
+
+Selection
+-------------------------
+Once you have everything set up as described above, you can do an active learning
+iteration by specifying the following three things in your Lightly docker config:
+
+- `method`
+- `active_learning.task_name`
+- `active_learning.score_name`
+
+Here's an example of how your config could look like if you want to add `100` more
+images to your dataset by doing active learning:
+
+
+.. tabs::
+
+    .. tab:: Run Command
+
+        .. code-block:: bash
+
+            docker run --gpus all --rm -it \
+                -v {OUTPUT_DIR}:/home/output_dir \
+                lightly/sampling:latest \
+                token=MYAWESOMETOKEN \
+                enable_corruptness_check=True \
+                remove_exact_duplicates=True \
+                pretagging=False \
+                pretagging_debug=False \
+                method='coral' \
+                stopping_condition.n_samples=100 \
+                stopping_condition.min_distance=-1 \
+                scorer_config.frequency_penalty=0.25 \
+                scorer_config.min_score=0.9 \
+                active_learning.task_name='my-classification-task' \
+                active_learning.score_name='uncertainty_margin' \
+                datasource.dataset_id={DATASET_ID} \
+                datasource.process_all=True
+
+
+    .. tab:: JSON
+
+        .. code-block:: javascript  
+
+            {
+                enable_corruptness_check: true,
+                remove_exact_duplicates: true,
+                enable_training: false,
+                pretagging: false,
+                pretagging_debug: false,
+                method: 'coral',
+                stopping_condition: {
+                    n_samples: 100,
+                    min_distance: -1
+                },
+                scorer: 'object-frequency',
+                scorer_config: {
+                    frequency_penalty: 0.25,
+                    min_score: 0.9
+                },
+                active_learning: {
+                    task_name: 'my-classification-task',
+                    score_name: 'uncertainty_margin'
+                },
+                datasource: {
+                    process_all: true
+                }
+            }
+
+Here, we set the `method` to `coral` which simultaneously considers the diversity
+and the active learning scores of the samples. The stopping condition was set the
+`n_samples: 100` and under `active_learning.task_name` we entered the task name of
+our predictions (see :ref:`TODO`). For this
+iteration, we picked the `uncertainty_margin` score. Learn more about the different scores in the next section.
+
+
+Active Learning with Custom Scores (deprecated)
+-----------------------------------------------
+
```

Comment:
Maybe add a date (e.g. is not recommended anymore as of March 2022 and will be deprecated)